pkg/provisioning/test/pipelines1/pipelines.go

```diff
@@ -77,8 +77,8 @@ var P1C2 = &connector.Instance{
 }
 
 var P1P1 = &processor.Instance{
-	ID:   "pipeline1:proc1",
-	Type: "js",
+	ID:     "pipeline1:proc1",
+	Plugin: "js",
```

Comment:
We should probably change the test pipeline yaml files to use the `plugin` field. Maybe create one with `type` that tests backwards compatibility.